libraries/cyclestreets-view/src/main/java/net/cyclestreets/AccountDetailsActivity.java

```diff
@@ -77,8 +79,38 @@ public void onCreate(final Bundle saved) {
     setupView();
   }
 
+  @Override
+  protected void onPause() {
+    super.onPause();
```

Comment:
What do you think about adding something like the following comment in onPause and onBackPressed:
"If successfully registered, clear user details."
Maybe put that common bit of code into a separate method along with the comment, to avoid repetition.

Reply:
If there is a success response from the API, then we know it can be cleared. Definitely avoid putting that work on the user.

Reply:
Are you referring to the PREF_PENDING_KEY comment above?

Reply:
Hi Tim.

Plus the 3rd comment I made:

> What do you think about adding something like the following comment in onPause and onBackPressed: "If successfully registered, clear user details." Maybe put that common bit of code into a separate method along with the comment, to avoid repetition.